UnityProject/Assets/Scripts/Effects/FloorEffect/LeaveFootprints.cs

```diff
@@ -63,6 +63,7 @@ public override bool WillAffectPlayer(PlayerScript eventData)
 				foreach (var feetSlot in playerStorage.GetNamedItemSlots(NamedSlot.feet))
 				{
 
+					if (feetSlot.ItemObject == null) continue;
```

Comment:
barefoot don't leave footprints?

Reply:
yeah, there's no reagent container to do it since the script that leaves footprints lives on the shoes